resources/profiles/Peopoly/machine/Peopoly Magneto X.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "machine_model",
+    "name": "Peopoly Magneto X",
+    "model_id": "Peopoly-Magneto-X",
+    "nozzle_diameter": "0.4",
+    "machine_tech": "FFF",
+    "family": "Peopoly",
+    "bed_model": "magnetox_model.stl",
+    "bed_texture": "magnetox_model_texture.png",
+    "hotend_model": "",
+	"default_materials": "Peopoly Generic PLA"

```

Comment:
While having only one default filament isn't wrong, I recommend listing all common filaments here. 
This way, they will be pre-selected when adding this machine.

Reply:
Thank you for your advice,
we will add it in the next PR.